src/tests/D3MHub.t.sol

```diff
@@ -978,285 +901,214 @@ contract D3MHubTest is DssTest {
 
-        assertRevert(address(d3mHub), abi.encodeWithSignature("cull(bytes32)", ilk), "D3MHub/unauthorized-cull");
+        assertRevert(address(hub), abi.encodeWithSignature("cull(bytes32)", ilk), "D3MHub/unauthorized-cull");
     }
 
     function test_no_cull_already_culled() public {
         _windSystem();
-        d3mHub.cage(ilk);
+        hub.cage(ilk);
 
-        d3mHub.cull(ilk);
-        assertRevert(address(d3mHub), abi.encodeWithSignature("cull(bytes32)", ilk), "D3MHub/already-culled");
+        hub.cull(ilk);
+        assertRevert(address(hub), abi.encodeWithSignature("cull(bytes32)", ilk), "D3MHub/already-culled");
     }
 
     function test_no_cull_no_ilk() public {
-        assertRevert(address(d3mHub), abi.encodeWithSignature("cull(bytes32)", bytes32("fake-ilk")), "D3MHub/pool-live");
+        assertRevert(address(hub), abi.encodeWithSignature("cull(bytes32)", bytes32("fake-ilk")), "D3MHub/pool-live");
     }
 
     function test_uncull() public {
         _windSystem();
-        d3mHub.cage(ilk);
+        hub.cage(ilk);
 
-        d3mHub.cull(ilk);
-        (uint256 pink, uint256 part) = vat.urns(ilk, address(d3mTestPool));
+        hub.cull(ilk);
+        (uint256 pink, uint256 part) = vat.urns(ilk, address(pool));
         assertEq(pink, 0);
         assertEq(part, 0);
-        assertEq(vat.gem(ilk, address(d3mTestPool)), 50 * WAD);
+        assertEq(vat.gem(ilk, address(pool)), 50 * WAD);
         uint256 sinBefore = vat.sin(vow);
-        (, , , uint256 culled, ) = d3mHub.ilks(ilk);
+        (, , , uint256 culled, ) = hub.ilks(ilk);
         assertEq(culled, 1);
 
         vat.cage();
-        d3mHub.uncull(ilk);
+        hub.uncull(ilk);
 
-        (uint256 ink, uint256 art) = vat.urns(ilk, address(d3mTestPool));
+        (uint256 ink, uint256 art) = vat.urns(ilk, address(pool));
         assertEq(ink, 50 * WAD);
         assertEq(art, 50 * WAD);
-        assertEq(vat.gem(ilk, address(d3mTestPool)), 0);
+        assertEq(vat.gem(ilk, address(pool)), 0);
         // Sin should not change since we suck before grabbing
         assertEq(vat.sin(vow), sinBefore);
-        (, , , culled, ) = d3mHub.ilks(ilk);
+        (, , , culled, ) = hub.ilks(ilk);
         assertEq(culled, 0);
     }
 
     function test_no_uncull_not_culled() public {
         _windSystem();
-        d3mHub.cage(ilk);
+        hub.cage(ilk);
 
         vat.cage();
-        assertRevert(address(d3mHub), abi.encodeWithSignature("uncull(bytes32)", ilk), "D3MHub/not-prev-culled");
+        assertRevert(address(hub), abi.encodeWithSignature("uncull(bytes32)", ilk), "D3MHub/not-prev-culled");
     }
 
     function test_no_uncull_mcd_live() public {
         _windSystem();
-        d3mHub.cage(ilk);
+        hub.cage(ilk);
 
-        d3mHub.cull(ilk);
+        hub.cull(ilk);
 
-        assertRevert(address(d3mHub), abi.encodeWithSignature("uncull(bytes32)", ilk), "D3MHub/no-uncull-normal-operation");
-    }
-
-    function test_quit_culled() public {
```

Comment:
I know these two are not actually test anything in the `Hub` itself, but they were kind of important to prove we can move the position. Do you think we can actually keep them? Maybe removing the pool part.

Reply:
These tests should be re-added as integration tests imo. Having them here doesn't really show us anything.

Reply:
I think it would be better to leave these in, at least until we have the same coverage in each integration test.
Apart from actually testing stuff, they serve as a "guide" on how these sequences look like.

(speaking about test_quit_culled, test_quit_not_culled, test_pool_upgrade_quit, test_plan_upgrade, test_hub_upgrade_same_d3ms, testFail_hub_upgrade_kills_old_hub, test_hub_upgrade_new_d3ms))

Reply:
Ok, I'll re-add.

Reply:
Ok it's been re-added.